docs/autorenew-feature.md

```diff
@@ -0,0 +1,71 @@
+# The autorenew feature
+
+## Goal
+-	Allow Hedera Services to automatically search for entities such as accounts, files, smart contracts, topics, tokens... that are about to expire and renew them.
+
+## Design
+- Introduce new settings in `application.properties`:
+  * `autorenew.isEnabled`
+  * `autorenew.numberOfEntitiesToCheck`
+  * `autorenew.maxNumberOfEntitiesToRenew`
+  * `autorenew.gracePeriod`
+- Each Hedera entity has an `expirationTime` which is the effective consensus timestamp at (and after) which the entity is set to expire.
```

Comment:
Not all entities have a field named `expirationTime`. Some name it `expiry` like the HTS services. Since no one should be using these fields can we take the time now to rename them to make them consistent? Or maybe add a new field and deprecate the old?

Reply:
Yes, we are aware of this inconsistency. Will create an issue for consolidation.

Reply:
#1197